src/__test__/manifest.loader.test.ts

```diff
@@ -0,0 +1,98 @@
+import o from 'ospec';
+import { Manifest } from '../manifest';
+import { isManifestsDifferent, ManifestLoader } from '../manifest.loader';
+
+o.spec('ManifestLoader', () => {
+  // Common test data (some manifests are used in different tests)
+
+  const manifestA: Manifest = {
+    path: '/media/paul/RGBi1',
+    size: 12141773487666,
+    files: [
+      {
+        path: 'RGBi Imagery/RGBI_Canterbury 0.3m Rural Aerial Photos (2015-16)/RGBI_BX16_5K_0607.tif',
+        size: 384096570,
+        hash: 'sha256-BT9aW9cqFR0P3rTknVjHDjPPPuVucu68RdEqEbMQ2L4=',
+      },
+    ],
+  };
+
+  const manifestB: Manifest = {
```

Comment:
NIT: lots of duplication here, could maybe just do

```
const manifestA = createManifest([{ path: ''RGBi Imagery/RGBI_Canterbury 0.3m Rural Aerial Photos (2015-16)/RGBI_BX16_5K_0607.tif'' }])
const manifestA = createManifest([{ path: 'RGBi_otago_rural_2017-2019_0.3m/2018_BZ13_5000_0901.tif'' }])
```

and just use the filename as the hash for it.